CONTRIBUTING.md

```diff
@@ -189,70 +390,122 @@ $ ./node ./test/parallel/test-stream2-transform.js
-When you push that branch to your fork, GitHub will automatically update the
-Pull Request.
+Please try to do your best at filling out the details but feel free to skip
+parts if you're not sure what to put.
 
-You can push more commits to your branch:
+Once opened, pull requests are usually reviewed within a few days.
+
+#### Step 9: Discuss and update
+
+You will probably get feedback or requests for changes to your Pull Request.
+This is a big part of the submission process so don't be discouraged! Some
+contributors may sign off on the pull request right away, others may have
+more detailed comments or feedback. This is a necessary part of the process
+in order to evaluate whether the changes are correct and necessary.
+
+To make changes to an existing Pull Request, make the changes to your local
+branch, add a new commit with those changes, and push those to your fork.
+GitHub will automatically update the Pull Request.
 
 ```text
 $ git add my/changed/files
 $ git commit
 $ git push origin my-branch
 ```
 
-Or you can rebase against master:
+It is also frequently necessary to synchronize your pull request with other
+changes that have landed in `master` by using `git rebase`:
 
 ```text
 $ git fetch --all
 $ git rebase origin/master
 $ git push --force-with-lease origin my-branch
 ```
 
-Or you can amend the last commit (for example if you want to change the commit
-log).
+**Important:** The `git push --force-with-lease` command is one of the few ways
+to delete history in `git`. Before you use it, make sure you understand the risks.
+If in doubt, you can always ask for guidance in the Pull Request or on
+[IRC in the #node-dev channel](https://webchat.freenode.net?channels=node-dev&uio=d4).
+
+If you happen to make a mistake in any of your commits, do not worry. You can
+amend the last commit (for example if you want to change the commit log).
 
 ```text
 $ git add any/changed/files
 $ git commit --amend
 $ git push --force-with-lease origin my-branch
 ```
 
-**Important:** The `git push --force-with-lease` command is one of the few ways
-to delete history in git. Before you use it, make sure you understand the risks.
-If in doubt, you can always ask for guidance in the Pull Request or on
-[IRC in the #node-dev channel](https://webchat.freenode.net?channels=node-dev&uio=d4).
+There are a number of more advanced mechanisms for managing commits using
+`git rebase` that can be used but are beyond the scope of this guide.
 
 Feel free to post a comment in the Pull Request to ping reviewers if you are
 awaiting an answer on something. If you encounter words or acronyms that
 seem unfamiliar, refer to this
 [glossary](https://sites.google.com/a/chromium.org/dev/glossary).
 
-Note that multiple commits often get squashed when they are landed (see the
-notes about [commit squashing](#commit-squashing)).
+##### Approval and Request Changes Workflow
+
+All pull requests require "sign off" in order to land. Whenever a contributor
```

Comment:
Should / does it make sense to have capitalization of PRs be standardized across the guide? Earlier in the guide it appears as Pull Request.